Dockerfile

```diff
@@ -477,8 +495,22 @@ LABEL org.apache.airflow.distro="debian" \
   org.apache.airflow.version="${AIRFLOW_VERSION}" \
   org.apache.airflow.uid="${AIRFLOW_UID}" \
   org.apache.airflow.gid="${AIRFLOW_GID}" \
-  org.apache.airflow.mainImage.buildId=${BUILD_ID} \
-  org.apache.airflow.mainImage.commitSha=${COMMIT_SHA}
+  org.apache.airflow.mainImage.buildId="${BUILD_ID}" \
```

Comment:
FYI. GCR uses standard OCI `org.opencontainers.image.source` label to link image to the repository. This is an interesting idea and seems to work well. I used the opportunity to add all predefined OCI labels so our image is following the standards. That includes human-readable descriptions, versions, URLs etc. I still leave the "org.apache." ones for backwards compatibility.